karma.conf.js

```diff
@@ -0,0 +1,85 @@
+// Karma configuration
+// Generated on Fri May 06 2016 17:58:51 GMT+0530 (IST)
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
+
+
+    // list of files / patterns to load in the browser
+    files: [
+        './bower_components/angular/angular.js',
+        './bower_components/angular-mocks/angular-mocks.js',
+        './bower_components/leaflet/leaflet.*',
+        './bower_components/**/angular-simple-logger.js',
+        './bower_components/**/angular-animate.js',
+        './bower_components/**/angular-aria.js',
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533145513/issues/source?bid=3233857&fileBranchId=3284028#l23)